src/main/java/com/example/demo/model/UrlRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.example.demo.model;
+
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class UrlRequest {
+    private int id;
```

Comment:
I think we don't need an id here and the expiration date as well. Instead of it, we can have `startDate` or smth similar and will count that field as a start of our shortUrl TTL(time to live)

Reply:
I deleted the id and the expiration data attributes for the request. The Entity is currently saving the creationDate / startDate